### DISCUSSION TITLE
Feature Request: AVIF image format support

### DISCUSSION THREAD

**Discussion by shrekuu** [spatie/image-optimizer#189] [Ideas]
Hi dear Spatie team:

Do you mind adding AVIF image format support? 

https://caniuse.com/?search=avif

I see AVIF images are much smaller than WEBP images.

(I am building an image processing app for my app. It serves images of different formats according to  this`Accept:
image/avif,image/webp,image/apng,image/svg+xml,image/*,*/*;q=0.8` from the request header.)

**Comment by freekmurze** [comment]
If this is easy to add, feel free to send a PR

Make sure to include tests and an updated readme.
